### ISSUE TITLE
meds tool should use updated ATC hierarchy

### ISSUE THREAD

**Issue by lauraevans** [TriNetX/dq-tools#9]

**Comment by LoraLingrey-TriNetX** [comment]
@lauraevans - Does this relate to the issue that was listed in the feature request table (row 4)?

**Comment by lauraevans** [comment]
Yes, same issue - because it's static it's stuck at the version of the meds hierarchy that was in place when it was first built
